src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -107,6 +108,12 @@ private OrbitInfo GetNextPatch()
             return ! GetHasNextPatch() ? null : new OrbitInfo(orbit.nextPatch,Shared);
         }
 
+
+        private double GetNextPatchETA()
+        {
+            return  ! GetHasNextPatch() ? Double.PositiveInfinity : orbit.EndUT - Planetarium.GetUniversalTime();
```

Comment:
I'd prefer a negative number.  a negative number *CANNOT* be a valid ETA time, while a zero could be valid, for just an instant, hypothetically.